src/backend/base/langflow/custom/custom_component/component.py

```diff
@@ -17,6 +17,7 @@
 from langflow.helpers.custom import format_type
 from langflow.schema.artifact import get_artifact_type, post_process_raw
 from langflow.schema.data import Data
+from langflow.schema.dotdict import dotdict  # noqa: TCH001
```

Comment:
Did you run into an issue putting this in a `TYPE_CHECKING` block?

Reply:
Moved to typechecking block